plugin.py

```diff
@@ -241,6 +211,47 @@ def on_result(self, payload: str) -> None:
 })
 
 
+def runnable_to_terminus_args(runnable: Runnable, view: sublime.View) -> Dict[str, Any]:
+    """
+    Convert the data of a Runnable into the arguments of a "terminus_open" command.
+    """
+    args = runnable["args"]
+    if args["overrideCargo"]:
+        cargo_path = args["overrideCargo"]
+    else:
+        cargo_path = "cargo"
```

Comment:
I removed the `shutil.which` code here because Terminus will just tell us if the executable is found or not.